helmfile/values/prometheus-user-alerts-wc.yaml.gotmpl

```diff
@@ -19,5 +19,8 @@ defaultRules:
     # Rook is handeled by the cluster operators. Users would normally not care
     # about these alerts, but we have no other way of gathering them currently.
     rookMonitor: {{ .Values.monitoring.rook.enabled }}
+    {{- if not .Values.prometheus.predictLinearLimit }}
```

Comment:
Feels a bit strange that we use predictLinearLimit to describe the limit and disabling it. We usually do not do that.

Reply:
yes it is, I will see if I can add the enabled flag instead.

Reply:
let me know if it looks better now

Reply:
I think it looks good, but we need a migration script for the change.

Reply:
yes I will create that. I wanted some feedback before creating the migration script

Reply:
added the migration script